shared/src/main/scala/pl/metastack/metaweb/dsl/package.scala

```diff
@@ -5,6 +5,20 @@ import pl.metastack.metaweb.tag.HTMLTag
 
 package object dsl {
   implicit class NodeRefDSLExtensions(nodeRef: NodeRef[HTMLTag]) {
```

Comment:
Btw: if we make this conversion a part of NodeRef, the user will not need to import anything to use it.